charts/rasa-x/templates/_helpers.tpl

```diff
@@ -131,3 +131,21 @@ Return the port of the action container.
 {{- define "rasa-x.custom-actions.port" -}}
 {{- default 5055 .Values.app.port -}}
 {{- end -}}
+
```

Comment:
These two defines allow for setting different extra environment variables for the rasa/rasa-x deployments.  They are used in `rasa-deployments.yaml` and `rasa-x-deployment.yaml` respectively and there are examples of defining extra env vars in the `values.yaml` file.